suite-common/message-system/config/config.v1.json

```diff
@@ -54,29 +54,29 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": ">=24.5.2",
+                        "desktop": "<=24.7.1",
                         "mobile": "!",
-                        "web": ">=24.5.2"
+                        "web": "<=24.7.1"
```

Comment:
probably should never occur, `!` is enough

Reply:
Resolved